ddp-workspace/projects/ddp-sdk/src/lib/services/authentication/auth0Adapter.service.ts

```diff
@@ -11,10 +11,11 @@ import { Auth0Mode } from '../../models/auth0-mode';
 import { AnalyticsEventCategories } from '../../models/analyticsEventCategories';
 import { AnalyticsEventActions } from '../../models/analyticsEventActions';
 import { JwtHelperService } from '@auth0/angular-jwt';
-import { Subject } from 'rxjs';
-import { takeUntil, skip, take } from 'rxjs/operators';
+import { bindNodeCallback, Observable, Subject } from 'rxjs';
+import { takeUntil, skip, take, tap, catchError, mergeMap } from 'rxjs/operators';
```

Comment:
Please remove unused `skip` and `catchError`

Reply:
6f39484